Makefile

```diff
@@ -1,10 +1,12 @@
-# El version y nombre son los que figuran en el package.json.
-VERSION=$(shell scripts/obtenerVersion.sh)
-# El NOMBRE lo usa el empaquetador para crear archivos y carpetas y para darle nombre a los binarios
-NOMBRE=$(shell scripts/obtenerNombre.sh)
-# La ruta del ejecutable de ember
+# The ember executable path.
 EMBER=./node_modules/.bin/ember
 
+#The project name, used by electron-packager in order to create files and folders for the app binaries.
+NAME=$(shell node -p "require('./package.json').name")
+
+# The project version.
+VERSION=$(shell node -p "require('./package.json').version")
+
```

Comment:
It's ok if you want to use node instead of sed to get app version and name. But I'd put this inside the scripts `obtenerVersion.sh` and `obtenerNombre.sh`, because we need to be consistent in the method for getting app version. (those scripts are used in other places). Let's not introduce an inconsistency.
We could erase those scripts and use node everywhere if you prefer, but let's get version and name in every place in the same way.

Places where obtenerVersion is used: Makefile, .travis.yml, and pilas-engine-bloques-website/extras/obtener_links.py
Places where obtenerNombre is used: Makefile, .travis.yml